.gitignore

```diff
@@ -1,4 +1,357 @@
-.vs
-bin
-obj
+## Ignore Visual Studio temporary files, build results, and

```

Comment:
Can we not do this?
This `.gitignore` seems too much.
There are c++, python, javascript, tfs ignores here, and nothing of that ever going to be used here.